frontend/src/components/pages/StudentAssessment.tsx

```diff
@@ -0,0 +1,182 @@
+/* eslint-disable react/prop-types */
+import React from "react";
+import { ChakraProvider, Box, Text, Stack, Button } from "@chakra-ui/react";
+import {
+  JUMP_MATH_LOGO,
+  MULTI_SELECT,
+  MULTI_CHOICE,
+  SHORT_ANSWER,
+} from "../../assets/images";
+import theme from "../../themes";
+
+type BoxProps = {
+  children: React.ReactNode;
+  top?: string;
+  left?: string;
+};
+
+const InnerBox: React.FC<BoxProps> = ({ children, top, left }) => (
+  <Box
+    backgroundColor="blue.50"
+    borderRadius="10px"
+    position="absolute"
+    padding="10px"
+    gap="40px"
+    width="115px"
+    height="115px"
+    top={top}
+    left={left}
+  >
+    {children}
+  </Box>
+);
+
+const OutterBox: React.FC<BoxProps> = ({ children, top, left }) => (
+  <Box
+    backgroundColor="rgba(232, 237, 241, 0.3)"
+    borderRadius="10px"
+    position="absolute"
+    padding="32px"
+    gap="40px"
+    width="480px"
+    height="376px"
+    top={top}
+    left={left}
+  >
+    {children}
+  </Box>
+);
+
+type AssessmentSummaryProps = {
+  numOfQuestions: number;
+  totalPoints: string;
+  questionTypes: string[];
+  rules: string;
+  questionText: string;
+  totalPointsText: string;
+  questionTypesText: string;
+  startTime: string;
+  test: string;
+};
+
+const AssessmentSummary = ({
+  numOfQuestions = 12,
+  totalPoints = "50 + 1 (Bonus)",
+  questionTypes = ["Multiple choice", "Multi-Select", "Short Answer"],
+  rules = "The test WILL be monitored so please close any windows before starting the test. \nYou will have 1 hour to complete this test. No aids are permitted. \nIf you need clarification or assistance, please raise your hand quietly and I will come to you. \nGood Luck! \n- Mr. Roberts",
+  questionText = "Number of Questions",
+  totalPointsText = "Total Points",
+  questionTypesText = "Question Types",
+  startTime = "Start Time: September 15, 2022 at 2:00pm",
+  test = "Unit 0 Review Test",
+}: AssessmentSummaryProps): React.ReactElement => {
+  return (
+    <ChakraProvider theme={theme}>
```

Comment:
this shouldn't be needed
```suggestion
```

Reply:
wiithout, everything turn to the original colours that the chakraprovider library has

Reply:
what do you mean by this? you should be able to override the colors

Reply:
for some reason, it isnt overiding. and if i deelte that line, all the code in the return statement becomes an error because there is no parent element then